Rock.Encryption/Symmetric/CryptoConfiguration.cs

```diff
@@ -12,6 +12,6 @@ public class CryptoConfiguration
         /// Gets the collection of credentials that will be available for encryption or
         /// decryption operations.
         /// </summary>
-        public List<Credential> Credentials { get; set; } = new List<Credential>();
+        public List<ICredential> Credentials { get; set; } = new List<ICredential>();
```

Comment:
I don't think this class will serialize if it depends on the interface. @JustinMoss: can you confirm?